org.jellyfin.JellyfinServer.yml

```diff
@@ -0,0 +1,325 @@
+      - --enable-chromaprint
+      - --enable-cuda
+      - --enable-cuda-llvm
+      - --enable-cuvid
+      - --enable-ffnvcodec
+      - --enable-gmp
+      - --enable-gnutls
+      - --enable-gpl
+      - --enable-libass
+      - --enable-libbluray
+      - --enable-libdav1d
+      - --enable-libdrm
+      - --enable-libfdk-aac
+      - --enable-libfontconfig
+      - --enable-libfreetype
+      - --enable-libfribidi
+      - --enable-libmp3lame
+      - --enable-libopus
+      - --enable-libplacebo
+      - --enable-libshaderc
+      - --enable-libsvtav1
+      - --enable-libtheora
+      - --enable-libvorbis
+      - --enable-libvpx
+      - --enable-libwebp
+      - --enable-libx264
+      - --enable-libx265
+      - --enable-libzimg
+      - --enable-libzvbi
+      - --enable-lto
+      - --enable-nvdec
+      - --enable-nvenc
+      - --enable-opencl
+      - --enable-shared
+      - --enable-vaapi
+      - --enable-version3
+      - --enable-vulkan
+      - --extra-libs=-lfftw3f
+    sources:
+      - type: archive
+        url: https://github.com/jellyfin/jellyfin-ffmpeg/archive/refs/tags/v5.1.2-9.tar.gz
+        sha256: 5eefa589d1fe7b8878f7fa4dd71a61b5902299e420cc9e0ef1a406d288feeeee
+      - type: shell
+        commands:
+          - for n in $(cat debian/patches/series); do if [[ $n != "#"* ]]; then patch -Np1 -i "debian/patches/$n" -d .; fi; done
+    modules:
+      - shared-modules/linux-audio/fftw3f.json
+      - name: chromaprint
+        buildsystem: cmake-ninja
+        config-opts:
+          - -DCMAKE_BUILD_TYPE=Release
+          - -DBUILD_SHARED_LIBS=ON
+          - -DBUILD_TOOLS=OFF
+          - -DBUILD_TESTS=OFF
+          - -DFFT_LIB=fftw3f
+        sources:
+          - type: archive
+            url: https://github.com/acoustid/chromaprint/releases/download/v1.5.1/chromaprint-1.5.1.tar.gz
+            sha256: a1aad8fa3b8b18b78d3755b3767faff9abb67242e01b478ec9a64e190f335e1c
+      - name: libass
+        config-opts:
+          - --enable-shared
+          - --disable-static
+          - --with-pic
+        sources:
+          - type: archive
+            url: https://github.com/libass/libass/releases/download/0.16.0/libass-0.16.0.tar.gz
+            sha256: fea8019b1887cab9ab00c1e58614b4ec2b1cee339b3f7e446f5fab01b032d430
+      - name: libbluray
+        config-opts:
+          - --enable-shared
+          - --disable-static
+          - --with-pic
+          - --disable-doxygen-doc
+          - --disable-doxygen-dot
+          - --disable-doxygen-html
+          - --disable-doxygen-ps
+          - --disable-doxygen-pdf
+          - --disable-examples
+          - --disable-bdjava-jar
+        sources:
+          - type: archive
+            url: https://download.videolan.org/pub/videolan/libbluray/1.3.2/libbluray-1.3.2.tar.bz2
+            sha256: 456814db9f07c1eecdef7e840fcbb20976ef814df875428bfb81ecf45851f170
+      - name: intel-mediasdk
+        only-arches:
+          - x86_64
+        buildsystem: cmake
```

Comment:
cmake-ninja doesn't work?

Reply:
I missed this one. It is now cmake-ninja